File: templates/tests-php/BREACH-006.php
<?php

namespace Compliance\Tests;

use PHPUnit\Framework\TestCase;

class SecurityClassificationTest extends TestCase
{
    public function testVerifySecurityClassification(): void
    {
        $config = Config::load();
        $this->assertNotEmpty($config->auditMetadata->classification);
    }
}

File: templates/tests-csharp/BREACH-006.cs
using Xunit;

namespace Compliance.Tests
{
    public class SecurityClassificationTests
    {
        [Fact]
        public void TestVerifySecurityClassification()
        {
            var config = Config.Load();
            Assert.False(string.IsNullOrEmpty(config.AuditMetadata.Classification));
        }
    }
}
